doc/tutorial/using_gpu.txt

```diff
@@ -126,7 +126,8 @@ the GPU object directly.  The following code is modified to do just that.
 
   rng = numpy.random.RandomState(22)
   x = shared(numpy.asarray(rng.rand(vlen), config.floatX))
-  f = function([], gpuarray.basic_ops.GpuFromHost(None)(tensor.exp(x)))
+  gx = x.transfer(None)  # Transfer variable to GPU
+  f = function([], tensor.exp(gx))
```

Comment:
This is not the right change.  The .transfer should replace what GpuFromHost was doing i.e.:
```
f =  function([], tensor.exp(x).transfer(None))
```